app/src/main/java/net/bible/android/view/activity/page/MenuCommandHandler.kt

```diff
@@ -220,20 +218,12 @@ constructor(private val callingActivity: MainBibleActivity,
                         .setTitle(callingActivity.getString(R.string.backup_backup_title))
                         .setMessage(callingActivity.getString(R.string.backup_backup_message))
                         .setNegativeButton(callingActivity.getString(R.string.backup_phone_storage)) { dialog, which ->
-                            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.Q) {
```

Comment:
Since this is removed, I think this needs testing on those android versions.

Reply:
It would indeed be good to roughly go through different android versions with emulator (from lollipop (5) to the latest models).